README.md

```diff
@@ -1 +1,143 @@
-# sql-commands-intro
\ No newline at end of file
+# SQL Commands Introduction
+This workshop will teach you how to use basic SQL - Structured Query Language. SQL is a standard for relational database systems, while MySQL and PostgreSQL are implementations of the standard.
+
+## Syntaxes
+psql CLI
+```
+\c <database name> # Connect to database
+\i <file path> # Load SQL build file into currently open database
+```
+
+## Getting Started
+Make sure you have installed and setup PostgreSQL.
```

Comment:
Add a link to installation instructions, [DWYL's learn ppostgres](https://github.com/dwyl/learn-postgresql)